doc/eggex.md

```diff
@@ -311,6 +311,21 @@ Flags and translation preferences together:
 
     / digit+ ; ignorecase ; python /  # could translate to (?i)\d+
 
+The following flags are currently supported:
+
+#### `i` / `reg_icase` (Ignore Case)
+
+**With** this flag, the match _is not_ case sensitive.
+
+**Without** this flag, the match _is_ case sensitive.
+
+#### `reg_newline` (Multiline)
+
+**With** this flag, `$` will match before a newline and `^` will match after a
```

Comment:
We should use the eggex names `%end` and `%start` here.

And I think it should be `dot` and `![abc]` to be clearer, since `!...` could mean something else